### PR TITLE
Update Rambler.ru.xml

### PR DESCRIPTION
**Pull request by koops76**
#11114

### CODE REVIEW

File: src/chrome/content/rules/Rambler.ru.xml [hunk 1]
```diff
@@ -1,74 +1,38 @@
...
-
-403 Forbidden:
-	cnt.rambler.ru
-	counter.rambler.ru
-	scounter.rambler.ru
-	tools.rambler.ru
+	403 Forbidden:
+		cnt.rambler.ru
+		counter.rambler.ru
+		scounter.rambler.ru
```
Comment (J0WI) [inline src/chrome/content/rules/Rambler.ru.xml]: You just removed them and their working test urls. Please revert this and re-activate them. You may add an exclusion for `$` that Travis only tests the custom test url.

Comment (koops76) [inline src/chrome/content/rules/Rambler.ru.xml]: oops

File: src/chrome/content/rules/Rambler.ru.xml [hunk 2]
```diff
@@ -1,74 +1,35 @@
...
 	<target host="www.rambler.ru" />
 	<target host="audio.rambler.ru" />
 	<target host="booking.rambler.ru" />
 	<target host="brain.rambler.ru" />
 	<target host="c.rambler.ru" />
 	<target host="c-champ.rambler.ru" />
-	<!--<target host="cnt.rambler.ru" />
-	<test url="http://cnt.rambler.ru/top100.scn" /> -->
+	<target host="cnt.rambler.ru" />
 	<target host="cntcerber.rambler.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rambler.ru.xml]: This host refuses to connect over http and https, so you can remove it.

File: src/chrome/content/rules/Rambler.ru.xml [hunk 3]
```diff
@@ -148,4 +75,18 @@ Refused connection:
...
+	<test url="http://counter.rambler.ru/top100.scn" />
+	<test url="http://dn.rambler.ru/static/browser.css" />
+	<test url="http://dn.rambler.ru/static/js/notice.js" />
+	<test url="http://rcounter.rambler.ru/rcounter/rcounter.min.js" />
+	<test url="http://rcounter.rambler.ru/rcounter/rcounter.topline.min.js" />
+	<test url="http://rcounter.rambler.ru/rcounter/clickmap.min.js" />
+	<test url="http://sync.rambler.ru/set?partner_id=audtd/" />
+	<test url="http://scounter.rambler.ru/top100.scn?4402382" />
+	<test url="http://topline.rambler.ru/new/latest/bundle.css" />
+	<test url="http://topline.rambler.ru/new/latest/bundle.js" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rambler.ru.xml]: some of them need an exclusion on `/$` (see Travis).

Comment (J0WI) [inline src/chrome/content/rules/Rambler.ru.xml]: Still needs exclusions.

**Pull request closed without merging**